cogs/birth.py

```diff
@@ -0,0 +1,25 @@
+import random
+import time
+
+from peewee import *
+from models.ants import Ants
+from models.colony import Colony
+from models.users import Users
+
+class BotCommand:
+    def __init__(self, client):
+        self.client = client
+
+    def aliases(self):
+        return ["birth", "egg"]
+
+    async def handle(self, message, command, arguments):
```

Comment:
This should have the annotation `@commands.command()`, and the method should be called either birth or egg. `@commands.command` takes in a parameter for aliases, you can see it being used in the Ping command.